packages/cli/src/runDeploy.ts

```diff
@@ -219,9 +220,9 @@ export async function runDeploy(opts: DeployOptions): Promise<WorldDeploy> {
     );
   }
 
-  console.log(deploymentInfo);
+  console.log({ ...deploymentInfo, contracts: undefined });
```

Comment:
Nit: i assume this will also print `contracts: undefined`, could we instead explicitly print `{ worldAddress, blockNumber }` ?